app/views/quotes/new.html.erb

```diff
@@ -69,8 +69,8 @@
             <h5 class="modal-title text-white text-shadow" id="addressSubmitModalLabel">Let's start looking for your address:</h5>
          </div>
          <form action="" id="AddressSearchModal">
-            <div class="modal-body d-flex justify-content-center">
-               <input id="address" type="textbox" placeholder="123 Street Rd, Port Moody" class="w-100">
+            <div class="modal-body d-flex justify-content-center pac-contianer">
```

Comment:
Typo: `pac-container`, which makes me think [this](https://github.com/rosadiaz/snow_queen_v2_rails/pull/2/files#diff-f859e2848b07324f808ce1a5ccd9fcbaR94) might not be necessary. What's with the odd name, btw? Looks like a googleism